src/java/fr/paris/lutece/plugins/workflow/modules/formspdf/web/task/FormsPDFTaskComponent.java

```diff
@@ -119,9 +127,15 @@ public String getDisplayConfigForm( HttpServletRequest request, Locale locale, I
                 String [ ] strClassName = name.split( ".name" );
                 StringBuilder strClassLocation = new StringBuilder( strClassName [0] );
                 strClassLocation.append( ".path" );
-                listTemplatePDF.addItem( AppPropertiesService.getProperty( strClassLocation.toString( ) ), AppPropertiesService.getProperty( name ) );
+                listTemplatePDF.addItem( 0, AppPropertiesService.getProperty( name ) );
```

Comment:
With the code above, there might be multiple properties that match the filter, and there is a risk of reusing the same identifier.
I think it can be considered that there is only one property (workflow-formspdf.template_pdf.default.*) that exists only to define the default template, the others will be added in database.